### DISCUSSION TITLE
Change Cover (add more logos)?

### DISCUSSION THREAD

**Discussion by L-Cabral** [joaomlourenco/novathesis#333] [Q&A]
Hello! I would like to know if it is possible to add the logo from the university where I did my thesis in the cover. I do not find that option in the template.

**Comment by joaomlourenco** [comment]
There is no support for that in the template.ight person/service.
And I don't know if you are allowed to do that… 

If you get me some official answer saying you can do that, I'll add support for it in the template.

I'm not sure who will give you the official answer…  I would start by asking in the "Academic Services" and I guess they will redirect you to the right person/service.
